tailwind.config.js

```diff
@@ -103,6 +97,13 @@ module.exports = {
     fontFamily: {
       sans: ['Roboto']
     },
+    borderRadius: {
```

Comment:
next time avoid moving whole block of code if there is no reason for that because this would lead to reviewer confusion and maybe some conflicts